### ISSUE TITLE
Unstable formatting for node description

### ISSUE THREAD

**Issue by murdos** [mapado/prettier-plugin-gherkin#31]
I finally found some time to report and create a minimal reproduction case for this regression since #24.
FYI this has prevented us to use recent versions since our CI does a `prettier --check`, and we also have a git hook that does a `prettier --format` on commit.

Prettier configuration:
```yaml
printWidth: 140
singleQuote: true
tabWidth: 2
useTabs: false
endOfLine: "lf"

plugins:
  - prettier-plugin-gherkin
```

**A line is always added in the middle of the description on each reformatting.**

### Initial content
```gherkin
Feature: A feature

  Rule: A rule
    Is el tourre ed al mandede ed tionraladéc aaaaaaaaaaa bbb en "Succès", azeaz le caertyyy ddddddd ttttttttttt. Si ce n est pas le cas, et Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit.

    Example: An example
      When Some action
      Then Some result
```

### First formatting
```gherkin
Feature: A feature

  Rule: A rule
    Is el tourre ed al mandede ed tionraladéc aaaaaaaaaaa bbb en "Succès", azeaz le caertyyy ddddddd ttttttttttt. Si ce n est pas le cas, et
    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad
    minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit.


    Example: An example
      When Some action
      Then Some result

```

### Second formatting
```gherkin
Feature: A feature

  Rule: A rule
    Is el tourre ed al mandede ed tionraladéc aaaaaaaaaaa bbb en "Succès", azeaz le caertyyy ddddddd ttttttttttt. Si ce n est pas le cas, et

    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad
    minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit.


    Example: An example
      When Some action
      Then Some result

```

### Third formatting
```gherkin
Feature: A feature

  Rule: A rule
    Is el tourre ed al mandede ed tionraladéc aaaaaaaaaaa bbb en "Succès", azeaz le caertyyy ddddddd ttttttttttt. Si ce n est pas le cas, et


    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad
    minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit.


    Example: An example
      When Some action
      Then Some result

```

**Comment by murdos** [comment]
NB: I'm also wondering why there are 2 empty lines between the end of the description of the rule, and the example.
I think one empty line should be enough.

**Comment by jdeniau** [comment]
Hi @murdos 
This should have been fixed in 3.1.1

**Comment by murdos** [comment]
Thanks @jdeniau !
